### PR TITLE
Fixes #1185 Add Expression Profile Button Not Implemented

### LINKED ISSUE #1185: Add Expression Profile Button Not Implemented
Add Expression button not implemented.
image

```
Application:
MoBi® 12.0.0

Action is not implemented yet

Stack trace:

at OSPSuite.Presentation.MenuAndBars.NotImplementedCommand.Execute()
   at OSPSuite.Presentation.UICommands.UICommandExtensions.ExecuteWithinExceptionHandler(IUICommand uiCommand, IEventPublisher eventPublisher, IChangePropagator changePropagator)
```

It has 3 sub-buttons that are implemented, but what do you do for the top level button

image

### PR DESCRIPTION
**Pull request by rwmcintosh**
Fixes #1185

# Description
A button that shouldn't cause a command to be run, but it does

## Type of change

Please mark relevant options with an `x` in the brackets.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires documentation changes (link at least one [user](https://github.com/Open-Systems-Pharmacology/docs) or [developer](https://github.com/Open-Systems-Pharmacology/developer-docs) documentation issue):
- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
- [ ] Other (please describe):

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Integration tests
- [ ] Unit tests
- [x] Manual tests 
- [ ] No tests required

# Reviewer checklist

Mark everything that needs to be checked before merging the PR.

- [ ] Check if the code is well documented
- [ ] Check if the behavior is what is expected
- [ ] Check if the code is well tested
- [ ] Check if the code is readable and well formatted
- [ ] Additional checks (document below if any)
- [ ] Check if documentation update issue(s) are created if the option `This change requires a documentation update` above is selected

# Screenshots (if appropriate):

# Questions (if appropriate):

### CODE REVIEW

**Comment by rwmcintosh** [comment]
Not as hard as I thought.

File: src/MoBi.Presentation/Repositories/MenuBarItemRepository.cs [hunk 1]
```diff
@@ -234,7 +234,7 @@ protected override IEnumerable<IMenuBarItem> AllMenuBarItems()
             .WithDescription(ToolTips.ModelingRibbon.CreateModule)
             .WithIcon(ApplicationIcons.Module);
 
-         yield return CreateMenuButton.WithCaption(AppConstants.Captions.ExpressionProfile)
+         yield return CreateSubMenu.WithCaption(AppConstants.Captions.ExpressionProfile)
```
Comment (msevestre) [inline src/MoBi.Presentation/Repositories/MenuBarItemRepository.cs:237]: I don't quite understand how this is done here. 
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.Presentation/Repositories/MenuBarItemRepository.cs#L178

this is the original code in PKSim. Why is is done different in MoBI, I do not know. But the  "AddMetaEnzyme" etc sub entries needs to be added to the main menu. Which I don't see here

**Review by msevestre — CHANGES_REQUESTED** [review]
Looks wrong to me. I do not see where the items are added to the main menu. Clearly it's done somewhere but I'd suggest to do it the same as in PK-Sim so that we do not have two different implementation for the same stuff

File: src/MoBi.Presentation/Repositories/MenuBarItemRepository.cs (hunk 1 above)
Comment (rwmcintosh) [reply src/MoBi.Presentation/Repositories/MenuBarItemRepository.cs:237]: Why is that sub-menu from PK-Sim built that way, compared to:

https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.Presentation/Repositories/MenuBarItemRepository.cs#L383

Where it's created in ButtonGroupRepository? What's the difference between them?

**Pull request merged**